tests-resources/projects/test-liferay-workspace-ee/build.gradle

```diff
@@ -0,0 +1 @@
+version = '1.0.0'
```

Comment:
Are these test resources shared between functional and integration tests?  If not, they we need to move this test resources into the integration-tests folder.

Reply:
Yes, the projects or files putting in “test-resources” will be shared between functional and integration